forge-gui/res/cardsfolder/upcoming/monster_manual_zoological_study.txt

```diff
@@ -0,0 +1,16 @@
+Name:Monster Manual
+ManaCost:3 G
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 1 G T | Origin$ Hand | Destination$ Battlefield | ChangeType$ Creature | ChangeNum$ 1 | SpellDescription$ You may put a creature card from your hand onto the battlefield.
```

Comment:
How is the stack description for this when you run it? You can pop it in the Discord if that's easier

Reply:
Stack Description:
Monster Manual (7) - Creston puts a creature card from their hand onto the battlefield.

Reply:
Thanks, I'll add that to my list. Good enough for now but we should add that "may"